Source/Atomic/Resource/Image.cpp

```diff
@@ -1211,6 +1216,91 @@ bool Image::SaveJPG(const String& fileName, int quality) const
         return false;
 }
 
+bool Image::SaveDDS(const String& fileName) const
+{

```

Comment:
instead of moving it we might want to keep SaveDDS on Image class for desktop builds for consistency and where increasing the runtime size doesn't matter so much.  Though, this does complicate the script bindings (somehow need per platform API bindings where they diverge).  

A preference would probably be to have SaveDDS on Image, though only available on desktop builds (so don't build crunch on Android/iOS/Emscripten)